common/service.go

```diff
@@ -132,6 +132,48 @@ type Service interface {
 // It can be used for launching service from tests, etc.
 func InitializeService(service Service, config ServiceConfig) (*RestServiceInfo, error) {
```

Comment:
Docstring format: Needs to start with the function name. Also, the docstring could be more meaningful here.